lib/active_record/pg_enum/4.1/arel_visit_integer.rb

```diff
@@ -0,0 +1,19 @@
+# Patch to Arel::Visitors::Visitor specific to Rails 4.1
+# Rails 4.2 doesn't require a patch, Rails 5.0 requires a different patch
+
+module Arel
+  module Visitors
+    class DepthFirst < Arel::Visitors::Visitor
+      alias :visit_Integer :terminal
+    end
+
+    class Dot < Arel::Visitors::Visitor
+      alias :visit_Integer :visit_String
+    end
+
+    class ToSql < Arel::Visitors::Visitor
+      alias :visit_Integer :literal
+    end
+  end
+end
```

Comment:
Monkeypatching Arel is _waaay_ outside the bounds of this gem

Reply:
Agreed.  I got a little overzealous.  No need to use Arel at all here.